Description: attempt to make a so
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 rpfits (2.23-1precise1) precise; urgency=low
 .
   * Initial release
Author: Gijs Molenaar <gijs@pythonic.nl>

---
--- rpfits-2.23.orig/linux64/GNUmakefile
+++ rpfits-2.23/linux64/GNUmakefile
@@ -9,7 +9,7 @@ RPARCH := linux
 # GNU Fortran and C compiler flags.
 FC     := gfortran
-FFLAGS := -g -O -fimplicit-none -Wall
-CFLAGS := -g -O -Wall
+FFLAGS := -g -O -fimplicit-none -Wall -fPIC
+CFLAGS := -g -O -Wall -fPIC
 RANLIB := ranlib
 
 LDFLAGS :=
